client/extensions/wp-super-cache/contents-tab.js

```diff
@@ -180,19 +188,19 @@ class ContentsTab extends Component {
 
 				<div>
 				{ ! isEmpty( get( wpcache, 'cached_list' ) ) &&
-					<CacheStats header="Fresh WP-Cached Files" files={ wpcache.cached_list } />
+					this.renderCacheStats( wpcache.cached_list, translate( 'Fresh WP-Cached Files' ), true, false )
```

Comment:
> Also a question...do you know if there's a way to access `siteId` in `connect` in `ContentsTab` component? It's already been made available by the HOC's `connect`, but isn't available as part of `ownProps` or anything.

The order of invocation is the killjoy, I think. On the [relevant lines](https://github.com/Automattic/wp-calypso/blob/63fd1165f6764ec02bc6ae30c57202b60fe4442c/client/extensions/wp-super-cache/contents-tab.js#L281-L284), `WrapSettingsForm( getFormSettings )` is first wrapped around `ContentsTab` before `connectComponent` is wrapped around the result -- props thus flow thru `connectComponent` first and only then pass thru the HOC. 

(We might consider trying to switch the order since duplicating `getSelectedSiteId` selectors is a bit redundant indeed.)